acceptance_tests/features/steps/notification_letter.py

```diff
@@ -0,0 +1,58 @@
+from datetime import timedelta, datetime
+from time import sleep
+
+import paramiko
+from behave import given, when, then
+from retrying import retry
+
+from acceptance_tests.features.environment import poll_database_for_iac
+from config import Config
+from controllers.collection_exercise_controller import create_and_execute_collection_exercise
+
+
+@given('a reporting unit has been enrolled in a survey')
+def reporting_unit_enrolled(context):
+    now = datetime.utcnow()
+    context.start = datetime.now()
+    dates = {
+        "mps": now + timedelta(seconds=5),
+        "go_live": now + timedelta(seconds=10),
+        "return_by": now + timedelta(days=10),
+        "exercise_end": now + timedelta(days=11),
+    }
+    # Create action rule
+    create_and_execute_collection_exercise('cb8accda-6118-4d3b-85a3-149e28960c54', '0718', 'Testing notification file',
+                                           dates, short_name='Bricks')
+
+
+@when('the survey goes live')
+def survey_is_live(context):
+    context.iac_code = poll_database_for_iac('cb8accda-6118-4d3b-85a3-149e28960c54', '0718')
+
+
+@then('the reporting unit will receive a letter')
+def letter_is_received(context):
+    content = get_file_after_time(context.start)
+    assert context.iac_code in content, content
+
+
+def get_file_after_time(start_of_test):
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    ssh.connect(hostname=Config.SFTP_HOST,
+                port=int(Config.SFTP_PORT),
+                username=Config.SFTP_USERNAME,
+                password=Config.SFTP_PASSWORD)
+    with ssh.open_sftp() as client:
+        return retrying_get_file_after_time(client, start_of_test)
+
+
+@retry(retry_on_result=lambda r: not r, wait_fixed=1000, stop_max_delay=60000)
+def retrying_get_file_after_time(client, start_of_test):
+    files = client.listdir_attr(Config.SFTP_DIR)
+    files = sorted(files, key=lambda f: f.st_mtime, reverse=True)
+    latest_file_attributes = files[0]
+    with client.open(f'{Config.SFTP_DIR}/{latest_file_attributes.filename}') as latest_file:
+        content = str(latest_file.read()) if datetime.fromtimestamp(
+            latest_file_attributes.st_mtime) > start_of_test else None
+        return content
```

Comment:
This can probably be dropped down to a single line `return str(...` instead of assigning then returning.